### ISSUE TITLE
Add Flutter label

### ISSUE THREAD

**Issue by theswerd** [WorldHealthOrganization/app#328]
We should add language labels for issues so we can navigate them in an easier way.

**Comment by theswerd** [comment]
@SamMousa can you add a flutter label?

**Comment by advayDev1** [comment]
I'm not sure I understand this. We've decided per this standup as a team that Flutter shall be the client framework going forward.  We aren't building another client framework in parallel.

So all client/Flutter tasks should be labeled 'client' (and the server ones, which don't yet exist 'server').

Let me know if I've missed something, or close if you agree.

**Comment by theswerd** [comment]
The flutter tag will allow us to have the choice to switch moving forward.

Switching from ionic to flutter we(I think you had to do this) had to go through the issues and close all of the ionic ones. By tagging issues “flutter”, we will be more adaptable in future iterations if we ever choose to switch.

**Comment by SamMousa** [comment]
I don't think thjs is needed, it's extra work now for possible savings later. Next time we switch it won't be 16 hours before a deadline. So the 30 minutes going over issues won't be an issue.

Also from a non-Eng perspective it will be hard to differentiate a new issue being flutter or just client.

**Comment by advayDev1** [comment]
Correct. Let's reconsider if/when we need to change frameworks again.
